src/components/aboutUs/style.ts

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+
+export const Components = styled.div`
+display: flex;
```

Comment:
Add 2 or 4 spaces, how we do it for all styles, or use pretter for fix code style